### PR TITLE
Extend and fix atm

### PR DESCRIPTION
**Pull request by fschlueter**
Hi,
two commits. One adding a new function and one potentially fixing a bug. Can you check please.
Cheers,

### CODE REVIEW

File: radiotools/atmosphere/models.py [hunk 1]
```diff
@@ -705,8 +705,8 @@ def _get_density(self, zenith, xmax):
 #         dxmax_geo = self._get_distance_xmax_geometric(xmax, observation_level=0)
 #         return self._get_density_for_distance(dxmax_geo)
 
-    def _get_density_for_distance(self, d, zenith):
-        h = get_height_above_ground(d, zenith)
+    def _get_density_for_distance(self, d, zenith, observation_level=0):
+        h = get_height_above_ground(d, zenith, observation_level)
```
Comment (cg-laser) [inline radiotools/atmosphere/models.py:709]: do I see it correctly that this change does not have any effect? `_get_density_for_distance` is an external function that is never called with a third argument, and the default value for `observation_level` in `observation_level` is anyway zero. But adding the argument explicit doesn't harm.

**Review by cg-laser — APPROVED** [review]
a better helper for 2D histograms is great! Thanks for adding.

**Comment by fschlueter** [comment]
> a better helper for 2D histograms is great! Thanks for adding.

From the context I assume the approval belongs to an other pull request?

File: radiotools/atmosphere/models.py (hunk 1 above)
Comment (fschlueter) [reply radiotools/atmosphere/models.py:709]: Hi,
yes exactly. Within the existing code this change changes nothing. If there is never the need to add an observation level to this calculation we might want no to add it (there is the function `get_density_for_distance` which has this argument anyway. 
Are you happy with the function I added?

**Comment by cg-laser** [comment]
> > a better helper for 2D histograms is great! Thanks for adding.
> 
> From the context I assume the approval belongs to an other pull request?

No you added a new histogram function in this pull request.

**Comment by fschlueter** [comment]
Ups that was not intended. Anyway...

**Pull request merged**